test/includes.spec.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+const _ = require('lodash')
+const Promise = require('bluebird')
+const Joi = require('joi')
+const Hapi = require('hapi')
+
+let server, buildServer, destroyServer, hh;
+
+const schema = {
+    type: 'brands',
+    attributes: {
+        code: Joi.string().min(2).max(10),
+        description: Joi.string(),
+        year: Joi.number()
+    }
+};
+
+const data = {
+    attributes: {
+        code: 'MF',
+        description: 'Massey Furgeson',
+        year: 2007
+    }
+};
+
+
+//TODO just done the validation, actual includes is remaining
```

Comment:
Yeah, you can't validate the include parameter without relationships defined as part of the schema.
Perhaps we can defer this until when we implement the include functionality.